lib/Echidna/Solidity.hs

```diff
@@ -104,16 +106,18 @@ removeJsonFiles dir =
         let path = dir </> file
         whenM (doesFileExist path) $ removeFile path
 
-addresses :: SolConf -> NE.NonEmpty AbiValue
```

Comment:
NonEmpty has different properties than Set, so I'm not sure.

Reply:
I wish there were a non-empty set, but Set is better for what we do here as we pick random elements and list/non-empty list has slow random access. We would need to convert lists to sets, that takes linear time which kills the performance during fuzzing.